ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/participant/util/LfEngineToApi.scala

```diff
@@ -204,6 +206,11 @@ object LfEngineToApi {
       cmdss.toSeq)
   }
 
+  def lfContractKeyToApiValue(
+      verbose: Boolean,
+      lf: KeyWithMaintainers[Lf.VersionedValue[Lf.AbsoluteContractId]]): Either[String, ApiValue] =
+    lfVersionedValueToApiValue(verbose, lf.key)
+
```

Comment:
The numerous `fold`s added in this PR that deal with `LfEngineToApi` all follow the same pattern:

```suggestion

  @throws[RuntimeException]
  private[digitalasset] def assertOrRuntimeEx[A](failureContext: String, ea: Either[String, A]): A =
    ea fold (e => throw new RuntimeException(s"Unexpected error when $failureContext: $e"), identity)
```

Reply:
Applied by https://github.com/digital-asset/daml/pull/1586/commits/7f201b4538ddaff248fa9dd727b68b7915e87547